### ISSUE TITLE
Adding links for H5Web JupyterLab/VSCode extensions in `Exploring and Visualising HDF5 files` section of the doc

### ISSUE THREAD

**Issue by loichuder** [h5py/h5py#2248]
Hello :wave: 

I am part of the team who developed H5Web, a web-based viewer for HDF5 files as part of the PaNOSC project (https://github.com/silx-kit/h5web). 

We released extensions for people to be able to browser HDF5 files with H5Web in JupyterLab (https://github.com/silx-kit/jupyterlab-h5web) and VSCode (https://github.com/silx-kit/vscode-h5web).

Since you advertise for [viewers and other tools in the h5py doc](https://docs.h5py.org/en/3.8.0/related_projects.html#exploring-and-visualising-hdf5-files), I think it may be beneficial for h5py users to also add links here to the JupyterLab extension and/or the VSCode extension.

What do you think ?

**Comment by loichuder** [comment]
This page was outdated (last update 3 years ago) but @t20100 indeed recently updated it to include the h5web projects. 

So up to you: if you think it is enough to have links through this page, I won't complain.

**Comment by takluyver** [comment]
Hi Loïc! I'm certainly open to adding more tools to this page, and I think the h5web plugins would fit nicely.

I'd like to avoid h5py's list getting as long as the PaNOSC one, though, because I think long lists of options are less useful for this kind of thing. We can clearly leave out unmaintained tools, e.g. HDF Compass hasn't had any commits in a few years. I'd also suggest we limit it to tools meant for generic HDF5, rather than tools for specific formats built on top of HDF5, like Nexus (not that those don't have value).

Perhaps more difficult, I'd also suggest that each tool we include should have some unique advantage. E.g. a Jupyterlab plugin or a terminal based viewer is clearly useful if you're working in that context already, but if someone wrote another Jupyterlab HDF5 viewer, we would (somehow) decide either to keep jupyterlab-h5web in the list or to replace it, rather than listing both.

**Comment by epourmal** [comment]
@loichuder, I see that H5Web is not listed with The HDF Group [software page](https://portal.hdfgroup.org/display/support/Software+Using+HDF5). Please ask THG helpdesk to add it there.

**Comment by t20100** [comment]
@epourmal yes we've seen this too and we wanted to ask to add it here, I'll ask them.

The list https://github.com/panosc-eu/panosc/blob/master/Work%20Packages/WP4%20Data%20analysis%20services/resources/hdf5-viewers.rst was made to review existing solution before working on h5web and it's not really maintained.
To me the best place to maintain an exhaustive list of HDF5 browser/viewer would be on the HDF's group web site, like for the full [software page](https://portal.hdfgroup.org/display/support/Software+Using+HDF5). @ajelenak what do you think about it? Then h5py documentation could link to it.

**Comment by takluyver** [comment]
FWIW, my aim here is to have a more curated (i.e. not exhaustive) list. That's more difficult, but I think also more useful to users.

**Comment by ajelenak** [comment]
@t20100 I agree and will see what can be done.
